### ISSUE TITLE
Your current performance on VOC and COCO

### ISSUE THREAD

**Issue by benjaminshuai** [delftrobotics/keras-retinanet#12]
Hi, Can I check with you how does your implementation performs on VOC 07 (trained with 07 or 07+12) as well as on COCO val.  Your information would be highly appreciated.

**Comment by hgaiser** [comment]
The plan is to let it finish training on MS COCO (2017) and then do the evaluation. Hopefully these results will more or less match what is mentioned in their paper and I intend to publish these results and the model on this repositories main page.

It is currently still training and the loss is still decreasing. I'm waiting for that to finish before doing the evaluations.

**Comment by hgaiser** [comment]
Reopening since this still needs to be addressed.

**Comment by hgaiser** [comment]
After many experiments and tests, the latest results are published on the main page. I am still trying to improve these results to more closely match the paper results, but for now I will consider this issue resolved.

ps. as of now there are no plans to post results for Pascal VOC or provide pretrained models. You're welcome to train a Pascal VOC model and make PR for the results / pretrained model.
